src/manipulation.js

```diff
@@ -26,7 +26,7 @@ define( [
 	dataPriv, dataUser, acceptData ) {
 
 var
-	rxhtmlTag = /<(?!area|br|col|embed|hr|img|input|link|meta|param)(([\w:-]+)[^>]*)\/>/gi,
+	rxhtmlTag = /<(?!area|br|col|embed|hr|img|input|link|meta|param)(([a-z][^\x20\t\r\n\f\/\0>]*)[^>]*)\/>/gi,
```

Comment:
Does this regex need the two negated `>` in a row? There's one in the modified part and one that was there originally.